utils/nctl/sh/scenarios-upgrades/upgrade_scenario_09.sh

```diff
@@ -163,46 +163,51 @@ function _step_05()
     done
 }
 
-# Step 06: Await 1 era.
+# Step 06: Await 1 era and then start node 6 with trusted hash within the allowed range
 function _step_06()
 {
-    log_step_upgrades 6 "awaiting next era"
-    nctl-await-n-eras offset='1' sleep_interval='5.0' timeout='180' node_id='2'
+    POST_UPGRADE_BLOCK_HASH=${1}
```

Comment:
```suggestion
    local POST_UPGRADE_BLOCK_HASH=${1}
```

Reply:
added in https://github.com/casper-network/casper-node/pull/3644/commits/7f5aecd791c8b22d3cbd70ade3327c49ff75f862